scripts/ansible/roles/linux/intel/vars/redhat/ootpa.yml

```diff
@@ -13,8 +13,10 @@ intel_sgx_package_dependencies:
   - "protobuf-devel"
   - "dkms"
 
+intel_dcap_packages_url: "https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/rhel8.1-server/sgx_linux_x64_driver_1.35.bin"
+intel_dcap_packages_directory: "/tmp/sgx_linux_dcap_packages"
 intel_dcap_packages:
-  - "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/libsgx-ae-qve-1.6.100.2-1.el8.x86_64.rpm"
-  - "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/libsgx-dcap-ql-1.6.100.2-1.el8.x86_64.rpm"
-  - "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/libsgx-dcap-ql-devel-1.6.100.2-1.el8.x86_64.rpm"
-  - "https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/libsgx-urts-2.9.101.2-1.el8.x86_64.rpm"
+  - "libsgx-ae-qve"
```

Comment:
How would you construct the full path to these with https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/rhel8.0-Server/

Reply:
I believe the intention is to download the packages, add them to a local yum repo, and then install the same packages using the package name. 

Everything would be bundled into the SGX driver version (in this case v1.35) and there is no control over the versions of the individual packages themselves. Is this the desired behaviour?

Reply:
Yes, @CyanDevs  that's correct. Since the packages are no longer uploaded individually, we download the package bundle, unarchive it, and add it to the local yum repo in `scripts/ansible/roles/linux/intel/tasks/redhat/sgx-packages-install.yml`

Reply:
Thanks for the explanation @rs-- and @CyanDevs